css/styles.css

```diff
@@ -0,0 +1,17 @@
+.body {
+  display: sticky;
```

Comment:
better to put the sticky property to the header, otherwise our body won't scroll?

Reply:
thank you